client/src/resources/lang/en_us.json

```diff
@@ -1,5 +1,7 @@
 {
     "language": "English (United States)",
+    "language:var.0": "English",
+    "language:var.1": "American",
```

Comment:
American English

Reply:
Good idea. I just put American cause I thought it was funny